### DISCUSSION TITLE
How to use?

### DISCUSSION THREAD

**Discussion by iL03t** [jordanbtucker/totk-no-loot-boxes#19] [Q&A] [answered]
I apologize. How do you activate the amiibo for yuzu? I'm running what I'm supposed to but not sure how to activate amiibo.

**Comment by jordanbtucker** [comment, marked as answer]
With Yuzu, you have to have the amiibo bin files. Where to get those is outside of the scope of this project. Then to go File > Load amiibo.

With Ryujinx, it has a built in amiibo emulator, so you don't need the bin files. You just go to Actions > Scan amiibo.
